autonomi/src/client/archive.rs

```diff
@@ -6,7 +6,11 @@
 // KIND, either express or implied. Please review the Licences for the specific language governing
 // permissions and limitations relating to use of the SAFE Network Software.
 
-use std::{collections::HashMap, path::PathBuf};
+use std::{
+    collections::HashMap,
+    path::{Path, PathBuf},
+    time::SystemTime,
```

Comment:
I think we need to use `SystemTime` from `wasmtimer` here to prevent wasm problems.

```rust
#[cfg(not(target_arch = "wasm32"))]
pub use std::time::SystemTime;
#[cfg(target_arch = "wasm32")]
pub use wasmtimer::std::SystemTime;
```

or actually rather use our `sn_networking/src/target_arch.rs` mechanism here.

Reply:
To assure compatibility between JS apps and rust apps, we should use the same type. 
I'll use u64 as time since UNIX_EPOCH.